src/api-service/__app__/onefuzzlib/proxy.py

```diff
@@ -222,8 +222,8 @@ def save_proxy_config(self) -> None:
             ),
             forwards=forwards,
             region=self.region,
-            instrumentation_key=os.environ.get("APPINSIGHTS_INSTRUMENTATIONKEY"),
-            telemetry_key=os.environ.get("ONEFUZZ_TELEMETRY"),
+            instance_telemetry_key=os.environ.get("APPINSIGHTS_INSTRUMENTATIONKEY"),
```

Comment:
Right now, whenever we grab these key from the environment, we need to remember and preserve the mapping `instance <-> instrumentation`, `microsoft <-> telemetry`.

I suggest renaming the env vars elsewhere, but in the meantime, maybe we should define wrappers to `os.environ.get`, e.g.:
```python
def get_microsoft_telemetry_key() -> str:
    return os.environ.get("ONEFUZZ_TELEMETRY")
```
We could also just exports constants.